crates/turbopack-node/js/src/ipc/evaluate.ts

```diff
@@ -49,7 +49,8 @@ export const run = async (
           const value = await getValue(ipc, ...msg.args);
           await ipc.send({
             type: "end",
-            data: value === undefined ? undefined : JSON.stringify(value),
+            data:
+              value === undefined ? undefined : JSON.stringify(value, null, 2),
```

Comment:
We pass the string to `serde_json` which includes the source frame (of the JSON). It looks nicer when the source frame is actually readable, instead of being a single line of JSON.